src/main/kotlin/io/emeraldpay/dshackle/upstream/ethereum/EthereumChainSpecific.kt

```diff
@@ -90,15 +90,6 @@ object EthereumChainSpecific : ChainSpecific {
         return EthereumLabelsDetector(reader, chain)
     }
 
-    override fun subscriptionTopics(upstream: GenericUpstream): List<String> {
-        val subs = if (upstream.getCapabilities().contains(Capability.WS_HEAD)) {
```

Comment:
We remove it, but now how can we get subscriptionTopics?

Reply:
It is here - io.emeraldpay.dshackle.upstream.ethereum.EthereumEgressSubscription#getAvailableTopics
I have found that this part war redundant and we can drop it. It was never used.